patterns/overlapped-images.php

```diff
@@ -40,7 +40,14 @@
 			<!-- /wp:group -->
 
 			<!-- wp:paragraph {"className":"is-style-text-subtitle"} -->
-			<p class="is-style-text-subtitle"><?php echo wp_kses_post( _x( '<strong>Fleurs</strong> is a flower delivery and subscription business. Based in the EU, our mission is not only to deliver stunning flower arrangements across but also foster knowledge and enthusiasm on the beautiful gift of nature: flowers.', 'Content of the overlap images and paragraph pattern', 'twentytwentyfive' ) ); ?></p>
+			<p class="is-style-text-subtitle">
+			<?php
+				printf(
+					/* translators: %1$s is the brand name, e.g., 'Fleurs'. */
+					esc_html_x( '%1$s is a flower delivery and subscription business. Based in the EU, our mission is not only to deliver stunning flower arrangements across but also foster knowledge and enthusiasm on the beautiful gift of nature: flowers.', 'twentytwentyfive' ),
+					'<strong>' . esc_html_x( 'Fleurs', 'Example brand name in pattern', 'twentytwentyfive' ) . '</strong>'
+				);
+				?>
```

Comment:
```suggestion
			<p class="is-style-text-subtitle">
			<?php
				printf(
					/* translators: $s is the brand name, e.g., 'Fleurs'. */
					esc_html_x( '$s is a flower delivery and subscription business. Based in the EU, our mission is not only to deliver stunning flower arrangements across but also foster knowledge and enthusiasm on the beautiful gift of nature: flowers.', 'twentytwentyfive' ),
					'<strong>' . esc_html_x( 'Fleurs', 'Example brand name in pattern', 'twentytwentyfive' ) . '</strong>'
				);
				?>
			</p>
```

Also note that I've added a closing p tag.